crates/witness-generator/Cargo.toml

```diff
@@ -23,19 +23,19 @@ tokio-util = "0.7.15"
 
 alloy-eips = { workspace = true }
 alloy-rpc-types-eth.workspace = true
-alloy-primitives = { workspace = true, features = ["serde"] }
 ef-tests.workspace = true
 reth-stateless.workspace = true
 reth-ethereum-primitives = { workspace = true, features = [
     "serde",
     "serde-bincode-compat",
 ] }
-reth-primitives-traits = { workspace = true, features = [
-    "serde",
-    "serde-bincode-compat",
-] }
-reth-chainspec.workspace = true
```

Comment:
Unused dependencies.

Reply:
In a later PR, we can add a lint for this in every crate, so that unused dependencies will fail the linter and subsequently stop compilation